src/hardware/cpu/registers/flags.rs

```diff
@@ -0,0 +1,44 @@
+use crate::hardware::cpu::Condition;
+use bitflags::bitflags;
+
+bitflags! {
```

Comment:
Maybe replace by a Flags struct with impl for modifying specific flag. (or public members to avoid writing impl)

Reply:
Yes, if you don't want to use crates I can do my own implementation.

Reply:
Implemented